stress_test/controller.py

```diff
@@ -100,9 +113,17 @@ def check_status(self):
         """
         logging.info('[Controller] Checking the status')
 
+        q = queue.Queue()
         common.command_exec_wrapper([self.status_hnd],
```

Comment:
OK, let's make a good deal here: 

`command_exec_wrapper` executes a command locally or remotely, depending on the existence or not of ssh connection. Since using an SSH client for every NSTAT component is commonplace (even if we want to connect to a local component, in which case we create an SSH connectio with the localhost), I would propose stop using this wrapper and directly call the "remote" branch of it, i.e. `util.netutil.ssh_run_command`. 

Do this refactoring in every call site of `command_exec_wrapper`.